mollie/magento2
# Payment fee is not correctly refreshing on quote when placing order

Payment fee's don't seem to be correctly working with the latest version. It should show up in the totals when placing the order, this currently does not seem to be working correctly.

How to reproduce 1:
1. Set a payment fee for Klarna
2. In the frontend try to pay with Klarna. The payment fee does not show up because mollie_payment_fee + tax is still 0 value on the quote. You will see that the payment fee is not added to the totals overview

How to reproduce 2:
1. Set a payment fee for Klarna
2. Place a order with Klarna and decline the payment in the PSP so you get redirected back to the shop. Your quote still has the payment fee set to it's quote
3. Instead of paying with Klarna, now pay with iDeal (make sure it has no payment fee set to the method).
4. Set the payment to complete
5. In the backend in the order line it now shows a payment fee which is not correct

Added fix:
This fix changes the quote data when collecting the payment fee.
